requirements/scenarios.md

```diff
@@ -26,26 +26,26 @@ However, they do wish to add attestations to the validity of the content.
 1. The Wabbit Networks company builds their `net-monitor` software.
       - As a result of the build, they produce an [OCI Image][oci-image], a Software Bill of Materials (`SBoM`) and to comply with gpl licensing, produce another artifact which contains the source (`src`) to all the gpl licensed projects.
       - The `SBoM` and `src` artifacts are created as reference types to the image, creating a graph of artifacts.
-      - Each of the artifacts are signed with the Notary v2 wabbit-networks key.
-1. The Wabbit Networks signed contents are pushed to a public OCI compliant registry.
+      - Each of the artifacts are signed with the wabbit-networks key.
+2. The Wabbit Networks signed contents are pushed to a public OCI compliant registry.
```

Comment:
`1.`?

Reply:
Sorry. It was automatically changed by the editor, not sure how to disable it, but I will fix it manually for now.